Code/Framework/AzCore/AzCore/O3DEKernelConfiguration.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/PlatformDef.h>
+
+#if defined(AZ_MONOLITHIC_BUILD)
+    #define O3DEKERNEL_API
+#else
+    #if defined(O3DEKernel_EXPORTS)
```

Comment:
nit: Preprocessor names should be all CAPS.

(Maybe this was discussed before and approved)

Reply:
This is the default value that CMake creates for us, based on the [DEFINE_SYMBOL property](https://cmake.org/cmake/help/latest/prop_tgt/DEFINE_SYMBOL.html). We can change it, but do we need to? It won't appear anywhere else except in this header.